src/styles/globals.css

```diff
@@ -710,16 +711,16 @@ input[type='number']::-webkit-outer-spin-button {
 
 .location-question-container .answer-form-container .location-filled .rrs__button {
   background-color: #ffd09c;
-  color: white;
+  color: #333;
 }
 
 .location-question-container .answer-form-container .location-filled .rrs__button .rrs__multiselect-label__badge {
-  color: #fff;
+  color: #333;
   border: 2px solid;
 }
 
 .location-question-container .answer-form-container .location-filled svg {
-  fill: #fff;
+  fill: #333;
```

Comment:
Why not use a variable here?

Reply:
good spot! we didn't think about that in typing moment. 
Is now Fixed 👍